projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -372,3 +372,589 @@ The applicant understands that the Network Steward is not a legal entity and no
 [198](https://pkt-insight.cjdns.fr/#/PKT/pkt/tx/6912729413f16888cacacb006de0c3b3b70a3ad703bedfdc0bf5a478eaee2c57)
 [199](https://pkt-insight.cjdns.fr/#/PKT/pkt/tx/fcd0c40dccd650a271c9311b5ce302019455f8c02445faae1dee9e90b811a13f)
 [200](https://pkt-insight.cjdns.fr/#/PKT/pkt/tx/ec9ca663ea47b50aebaaa5b2d064959111ab09635cb26ecbad6bcb884cd34a8b)
+
+#PKT website for review and notes. A request for feedback from the Network Stewards
+
+------------------------ BEGINNING OF PROJECT ------------------------
+
+{PAGE}
+#Homepage
+Menu
+Download whitepaper button on right-side of menu
+Featured: animated gif of nodes connecting around the world (a sphere earth) in a web-like fashion with packets moving between them. 
+Text headers says: “The world’s first network-based blockchain”
```

Comment:
This is a bit knit-picky, but if I'm thinking about this site from the perspective of people who understand networking, I think we can make the Hero image metaphor and the text header a little more pragmatic - maybe these data points start at a website and packets get split  through the network and color coded to indicate  encryption and then converge onto one node, the  "user's" node.

Also I believe the  text-header should focus on what the value proposition  is. Something more like "Cryptographically secure networks built on Blockchain."